knees4bees/whats-cookin
# Fix/refactor recipe repository

--->  ((are you in the right branch??))
## What does this PR do?
- Separates out DOM search from data manipulation in scripts.js file
- Builds out RecipeRepository class to include reusable functions for filtering/searching recipes
- Creates recipe instances on page load that are then passed into a new RecipeRepository instance
- Fix: Adds one line that was removed in previous merge that was blocking Recipe Detail view from displaying

## How should it be tested?
 - Reviewer can double check that search/filter functionality still works appropriately
